### ISSUE TITLE
Components are not accessible by screen readers.

### ISSUE THREAD

**Issue by Muhnad** [jamesplease/materialish#289]
the most of components are not accessible by screen readers, so we can revisit every component and check it and make it accessible.

I can help with that but we need to discuss the way to revisit components and how acceptable changes we can do to avoid any compatibility issue with current users.

**Comment by jamesplease** [comment]
Thanks for the issue @Muhnad ! We are _definitely_ open to making changes to make the components more accessible.

In general, my approach would be to value accessibility _over_ making breaking changes to the API. In other words, we are open to making breaking API changes if it makes the components more accessible.

**Comment by jamesplease** [comment]
Cross-referencing this with #253 . One of the goals before the v1.0.0 release is to improve accessibility.
